internal/common/os_name.bzl

```diff
@@ -26,10 +34,10 @@ def os_name(repository_ctx):
     """
     os_name = repository_ctx.os.name.lower()
     if os_name.startswith("mac os"):
-        return "darwin_amd64"
+        return OS_NAMES[0]
     elif os_name.find("windows") != -1:
-        return "windows_amd64"
+        return OS_NAMES[1]
     elif os_name.startswith("linux"):
-        return "linux_amd64"
+        return OS_NAMES[2]
     else:
         fail("Unsupported operating system: " + os_name)
```

Comment:
Maybe the reverse functions belong in here?

```
def is_windows(os_name):
    return os_name == OS_NAMES[1]
```

etc... so that we're not assuming os_name.find("windows") is the correct check in various spots

Reply:
I am not sure I follow. Could you provide some more detail?

Reply:
I cleaned it up. Opted for is_windows_os(rctx) to use in the various places needed so they don't need to check if .find("windows") anymore or use os_name(rctx)